app/templates/partials/templates/guidance-send-a-document.html

```diff
@@ -9,5 +9,4 @@ <h2 class="heading-medium">
     Download your document at: ((link_to_document))
   </p>
 </div>
-  Next, use the API to upload your document.<!--<p> Follow the instructions to send a document by email in the <a href="https://www.notifications.service.gov.uk/documentation">API client documentation</a>.
-</p>-->
+  Next, use the API to upload your document. Follow the instructions to send a document by email in the <a href="https://www.notifications.service.gov.uk/documentation">API documentation</a>.
```

Comment:
This makes sure the link doesn’t break:
```suggestion
  Next, use the API to upload your document. Follow the instructions to send a document by email in the <a href="{{ url_for('main.documentation') }}">API documentation</a>.
```